opengrok-web/src/main/webapp/WEB-INF/tags/project.tag

```diff
@@ -0,0 +1,37 @@
+<%--
+CDDL HEADER START
+
+The contents of this file are subject to the terms of the
+Common Development and Distribution License (the "License").
+You may not use this file except in compliance with the License.
+
+See LICENSE.txt included in this distribution for the specific
+language governing permissions and limitations under the License.
+
+When distributing Covered Code, include this CDDL HEADER in each
+file and include the License file at LICENSE.txt.
+If applicable, add the following below this CDDL HEADER, with the
+fields enclosed by brackets "[]" replaced with your own identifying
+information: Portions Copyright [yyyy] [name of copyright owner]
+
+CDDL HEADER END
+
+Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
+--%>
+<%@ tag import="org.opengrok.indexer.web.Prefix" %>
+<%@ tag import="org.opengrok.indexer.web.Util" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
```

Comment:
these look like dead URLs

Reply:
the joke here is that this does not point to an external URI but identifies the tag library on the class path:

from jstl-1.2.jar

```
  <description>JSTL 1.1 core library</description>
  <display-name>JSTL core</display-name>
  <tlib-version>1.1</tlib-version>
  <short-name>c</short-name>
  <uri>http://java.sun.com/jsp/jstl/core</uri>
```